client/src/elm/Translate.elm

```diff
@@ -15508,622 +15508,622 @@ translationSet trans =
 
         ResilienceMessageGrowth3Title ->
             { english = "Never underestimate yourself"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Ufite imbaraga zo kwita ku buzima bwawe"
             }
 
         ResilienceMessageGrowth3Paragraph1 ->
             { english = "In our daily life, we encounter challenges that make us forget what is important to us. Try to find time to think about what matters; is it your family, people you care about, your work/profession, or is it your well-being. Those are your strengths. Think whether you live a life that serve your values."
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Mu buzima bwa buri munsi duhura n'ibitunaniza bituma twibagirwa icy'ingenzi kuri twe. Gerageza gufata umwanya utekereze kubigufitiye umumaro; ese n'umuryango wawe, n'abantu witaho, n'umwuga wawe, cyangwa n'ubuzima bwiza? Izo nizo mbaraga zawe. Tekereza niba ubaho ubuzima buguhesha agaciro."
             }
 
         ResilienceMessageGrowth3Paragraph2 ->
             { english = "Remember: If it matters to you, you will find a way to do it."
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just ""
```

Comment:
> If no translation, remain with `Nothing`.

@anvmn here we had a conversation with Adam and the team, since they are choosing some sayings, and some do not exist in Kinyarwanda instead of having them in english on the Kinyarwanda app we should not have them in Kinyarwand and have them in English because that's where they exist.